tests/pyensign/test_subscriber.py

```diff
@@ -0,0 +1,68 @@
+import pytest
+from asyncmock import patch
+
+from pyensign.subscriber import Subscriber
+from pyensign.exceptions import UnknownTopicError
+
+
+class TestSubscriber:
+    """
+    Tests for the high-level Subscriber class.
+    """
+
+    @pytest.mark.parametrize(
+        "topics, kwargs, exception",
+        [
+            ("topic", {"client_id": "id"}, ValueError),
+            ("", {"client_id": "id", "client_secret": "secret"}, ValueError),
+        ],
+    )
+    def test_bad_params(self, topics, kwargs, exception):
+        with pytest.raises(exception):
+            Subscriber(*topics, **kwargs)
```

Comment:
Can you add a note here to explain what about these params are bad? I'm not sure I understand why this should not work

Reply:
Sure, the user has to pass in a topic and credentials for their API key (or have credentials in the environment, but the test is assuming no environment).